doc/man/repoq.1.rst

```diff
@@ -0,0 +1,182 @@
+.. vim: ft=rst sw=2 sts=2 et
+
+==========
+ **repoq**
+==========
+
+------------------------------------------------
+Output repository information for given products
+------------------------------------------------
+
+:Author: Roman Neuhauser <rneuhauser+repose@sigpipe.cz>
+:Date: Feb 04, 2016
+:Copyright: GPL-2.0
+:Version: 0.28
```

Comment:
Version should be generated by build system from git or .. by spec by sed for example